frontend/src/pages/LocalLoginPage/SignupForm.js

```diff
@@ -92,7 +104,7 @@ const SignupForm = ({ redirectTo }) => {
                 format={(value) =>
                   value
                     ? createSlug(value, {
-                        lang: 'fr',
+                        lang: navigator.language.substring(0, 2),
```

Comment:
`navigator.language` is only supported by [96% of browsers](https://caniuse.com/?search=navigator.language) so there's a risk it makes the browser crashes.
You should add a check. And maybe default to the app language (`process.env.REACT_APP_LANG`) ?

Reply:
Actually the check should be on the `navigator` object itself (https://caniuse.com/mdn-api_navigator)

Reply:
Oh wow, good to know!
`process.env.REACT_APP_LANG` is only available server-side though, right?
Maybe `navigator?.language?.substring(0, 2) || 'fr'` is an easy compromise then?

Reply:
No `process.env.REACT_APP_LANG` is available on the front, it's used just a few lines above :-)

Reply:
my bad :D changed it now